build.sbt

```diff
@@ -4,8 +4,11 @@ version := "1.0-SNAPSHOT"
 
 scalaVersion := "2.11.8"
 
+enablePlugins(GatlingPlugin)
+
 libraryDependencies += "io.gatling" % "gatling-core" % "2.2.0"
 libraryDependencies += "io.gatling" % "gatling-http" % "2.2.0"
-
+libraryDependencies += "io.gatling.highcharts" % "gatling-charts-highcharts" % "2.2.0"
```

Comment:
> Convey the object code using peer-to-peer transmission, provided you inform
other peers where the object code and Corresponding Source of the work are
being offered to the general public at no charge under subsection 6d.
 
A separable portion of the object code, whose source code is excluded from the
Corresponding Source as a System Library, need not be included in conveying the
object code work.

This means we need to integrate the license of https://raw.githubusercontent.com/gatling/gatling-highcharts/master/LICENSE as well...